jkube-kit/enricher/api/src/main/java/org/eclipse/jkube/kit/enricher/api/util/KubernetesResourceUtil.java

```diff
@@ -399,7 +399,7 @@ private static String extractImageUser(String image, String groupId) {
         String imageUser = name.getUser();
         String projectGroupId = groupId;
         if(imageUser != null) {
-            return imageUser;
+            return imageUser.replaceAll("[^a-zA-Z0-9-]", "");
```

Comment:
Was this ported from FMP?

Reply:
No, in FMP imageUser comes off as `null`. This is just taken from line 408(https://github.com/eclipse/jkube/issues/908#issuecomment-919123278)

Reply:
https://github.com/fabric8io/fabric8-maven-plugin/blob/6525f61c042ebacfe7c945f9d8361598a5c95e31/core/src/main/java/io/fabric8/maven/core/util/kubernetes/KubernetesResourceUtil.java#L417